components/automate-ui/src/app/modules/cds/enable-content-modal/enable-content-modal.component.html

```diff
@@ -0,0 +1,22 @@
+<chef-modal label="license-lockout-label" [locked]="modalLocked" [visible]="modalVisible" (closeModal)="closeModal()">
+  <div class="register-wrapper">
+
+    <h2 class="title">Welcome to Chef Automate Content Library!</h2>
+    <form class="token" [formGroup]="tokenForm" (ngSubmit)="onFormSubmit()" novalidate>
+      <div class="token-container">
+        <chef-form-field>
+          <label for="token">Enter Chef Cloud Token<span aria-hidden="true"></span></label>
```

Comment:
i know this stuff is gonna be changed later, but i couldn't help myself

```suggestion
          <label for="token">Chef Cloud Token<span aria-hidden="true"></span></label>
```